packages/go/dawgs/drivers/neo4j/transaction.go

```diff
@@ -253,7 +253,7 @@ func (s *neo4jTransaction) createNode(properties *graph.Properties, kinds ...gra
 	} else if result := s.Raw(statement, queryBuilder.Parameters); result.Error() != nil {
 		return nil, result.Error()
 	} else if !result.Next() {
-		return nil, graph.ErrNoResultsFound
+		return nil, fmt.Errorf("%w: %w", graph.ErrNoResultsFound, result.Error())
```

Comment:
This was frustrating during some of our debugging. Still not sold on returning a "not found" during creation, but for backwards compatibility, I'm wrapping the original error path as well as the potential additional error context from the result. Why `result.Error()` is nil until you call `result.Next()` is likely something I don't even want to know the answer to.